README.md

```diff
@@ -1,47 +1,65 @@
 # acp-tf-redis
 Terraform module to build redis elasticache clusters
 
-## Module Usage
-
-```
-module "fake_redis" {
-  source = "git::https://github.com/UKHomeOffice/acp-tf-redis?ref=v0.0.1"
-
-  name                       = "fake-redis"
-  environment                = "${var.environment}"
-  node_type                  = "cache.t2.small"
-  number_of_nodes            = 2
-  cidr_blocks                = "${values(var.compute_cidrs)}"
-  subnet_ids                 = ["${var.subnet_ids}"]
-  vpc_id                     = "${var.vpc_id}"
-}
-```
```

Comment:
I think this (or something like this) is still worth keeping around

Reply:
Yes, you're right. Restored